### DISCUSSION TITLE
Workspaces missing.

### DISCUSSION THREAD

**Discussion by xmontero** [nocodb/nocodb#6674] [General]
Hi!

In the docs I see there should be "workspaces":

![image](https://github.com/nocodb/nocodb/assets/5641621/dc420d51-b9a4-4267-9516-07592ecfca2d)

But I see this:

![image](https://github.com/nocodb/nocodb/assets/5641621/f2b72d9f-709c-4a49-bde0-54ef0116d0da)

(running via `docker run` on an SQLite file).

# Question

How can I see the workspaces?

**Comment by dstala** [comment]
@xmontero 

You can use workspaces in our hosted solution (technical preview is available for free to try it out. visit https://nocodb.com). For OSS, it is always a default WS; Workspaces make sense for companies working with multiple teams.

<img width="1440" alt="Screenshot 2023-10-12 at 12 42 20 AM" src="https://github.com/nocodb/nocodb/assets/86527202/369c6b77-7287-44aa-b62c-2d8b2f212aa0">

**Comment by xmontero** [comment]
I see.

I run a travel agency. Plus I have about 10 side-projects.

The approach would then be 1 docker instance per each separate project. Run a dozen docker containers, each mapping to a different storage.

Is that correct?

**Reply by o1lab** [reply]
Hey, thanks for trying us out and your comment made our team's day. 🎉 

You can create unlimited projects on NocoDB's open source instance. I believe you would not need 10 different instances. Some of our users has 50+ projects running simultaneously with 100+ users.

**Reply by xmontero** [reply]
Ah, right! I see...

Simply there are not "workspaces" but I may have "separate databases" = "separate projects", exactly as I would do in a MySQL instance: I have N DBs in a server, each DB serving a different project.

That's all I needed for now! That's perfect. Can't understand why I did not see this by myself!

BTW, congrats for the project @o1lab This will change the world!

**Reply by martinjokub** [reply]
Yeah... I spent maybe 15 hours trying to figure out why on my hosted nocodb there is no workspaces.

Different GPTs tried to fool me, use this parameter:
NC_MULTI_WORKSPACE_ENABLED: "true"

It should be there. Then On the link: https://docs.nocodb.com/workspaces/workspace-overview/ I found:

> The Open Source NocoDB includes a default workspace, and it does not allow the creation of additional workspaces.

It is not that clear that this goes to self hosted nocodb.

And Projects or Workspaces or whatever you want to call it does not exist. You just have option to create separate bases for different projects.
